CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # FL Accelerator Changelog
 
+##  October 2022 release
+
+We are excited to announce the release of the September iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
+
+Here are the new features.
+
+
+### Repository structure
+- Enabled CI/CD on the repo to ensure nothing breaks.
+
+### Provisioning
+- Decoupled the provisioning of the workspace, the orchestrator, and the silos.
```

Comment:
I think this one is internal details, maybe mention just:

- All bicep scripts can be used as standalone to adapt to your specific infrastructure needs.